### ISSUE TITLE
Enable ability to send multiple goals via txt file. Also allow the ability to cancel each goal and all goals during execution

### ISSUE THREAD

**Issue by AnrdyShmrdy** [AnrdyShmrdy/ezrassor_teleop#8]
There are two files which contain incomplete code related to this: send_goal_list_cancelWIP.py and send_goal_listWIP.py
The purpose of them was to showcase the ability to send a list of actions/goals contained in a txt file that is passed to them
The program can read and validate the actions/goals contained in the txt file and use them, so nothing needs to be done there.
The problems encountered with this were that actions/goals were not always executed sequentially when the file was run, as the program did not wait for a goal that was sent to be finished or cancelled
Furthermore the program doesn't account for what to do after an action/goal is cancelled mid execution
To summarize, here is the list of tasks to do for this:

- [x] #18
- [ ] #19
- [ ] #20
- [ ] #21
